src/connector/src/sink/iceberg/java/src/main/java/com/risingwave/nimtable/SparkIcebergCompaction.java

```diff
@@ -0,0 +1,25 @@
+package com.risingwave.nimtable;
+
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+
+import java.util.List;
+
+public class SparkIcebergCompaction {
+    public static void main(String[] args) {
+        String database = args[0];
+        String table = args[1];
+        SparkSession session = SparkSession.builder()
+                .config("spark.sql.extensions", "org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions")
+                .config("spark.sql.catalog.nimtable", "org.apache.iceberg.spark.SparkCatalog")
+                .getOrCreate();
+        List<Row> rows = session.sql(String.format("CALL nimtable.system.rewrite_data_files(table => '%s.%s', options => map('rewrite-all', 'true'))", database, table)).collectAsList();
+        System.out.printf("compaction success: %s/%s, output: %s%n", database, table, rows);
+        try {
+            List<Row> expireSnapshotOutputRows = session.sql(String.format("CALL nimtable.system.expire_snapshots(table => '%s.%s')", database, table)).collectAsList();
+            System.out.printf("expire_snapshots success: %s/%s, output: %s%n", database, table, expireSnapshotOutputRows);
```

Comment:
The default parameter of expiration `older_than` is  5 days ago. I am not sure whether it is a good choice for every table. Maybe we need to make it configurable or support a manual compaction method in the future
older_than | ️ | timestamp | Timestamp before which snapshots will be removed (Default: 5 days ago)
-- | -- | -- | --